control/eventMacros.txt

```diff
@@ -0,0 +1,45 @@
+
+automacro test_slot1 {
```

Comment:
del  control/eventMacros.txt

Reply:
It is there so people can see how this system would work.